cluster/gce/gci/configure.sh

```diff
@@ -264,6 +264,18 @@ runtime-endpoint: ${CONTAINER_RUNTIME_ENDPOINT:-unix:///var/run/dockershim.sock}
 EOF
 }
 
+function install-exec-auth-plugin {
+  if [[ ! ${EXEC_AUTH_PLUGIN_URL:-} ]]; then
+      return
+  fi
+  local -r plugin_url="${EXEC_AUTH_PLUGIN_URL}"
+  local -r plugin_sha1="${EXEC_AUTH_PLUGIN_SHA1}"
```

Comment:
Are you delivering these parameters via kube-env?

Reply:
Yes